index.html

```diff
@@ -302,13 +302,38 @@ <h2 class="mg-1-tb">Primary :</h2>
 				</iframe><br> -->
 
 				<div class="container-lg">
-					Outlined:
+					<!-- Outlined:
 					<button class="btn primary-outlined-btn">
 						Primary</button>
 					<button class="btn secondary-outlined-btn">
 						Secondary
-					</button>
-				</div><br />
+					</button> -->
+					<h1 class="mg-2-top mg-point6-bot title-lg hover-underline-animation">Solid Button</h1>
+					<div class="mg-1-bot">Lorem ipsum dolor sit amet consectetur, adipisicing elit. A quos sapiente ipsum doloremque cumque
+						saepe?</div>
+					<div class="cont-shadow component-container pd-1-all mg-1-bot ">
+						<button class="button">DEFAULT</button>
+						<button class="button btn-primary">PRIMARY</button>
+						<button class="button btn-secondary">SECONDARY</button>
+						<button class="button btn-success">SUCCESS</button>
+						<button class="button btn-warning">WARNING</button>
```

Comment:
use prettier to format the code, it will increase readability

Reply:
✅✅